packages/location-state-core/docs/API.md

```diff
@@ -301,3 +303,47 @@ Returns the following hooks to which `StoreName` is bound.
 export const { useLocationState, useLocationStateValue, useLocationSetState } =
   getHooksWith<"local" | "indexeddb">();
 ```
+
+## Syncer
+
+```ts
+type Syncer = {
+  key(): string | undefined;
+  sync(arg: { listener: (key: string) => void; signal: AbortSignal }): void;
+  updateURL(url: string): void;
+};
+```
+
+`Syncer` is an interface for synchronizing with history location. By implementing `Syncer`, you can customize how to synchronize with the history location.
+
+### class `NavigationSyncer`
+
+```ts
+export declare class NavigationSyncer implements Syncer {
+  constructor(navigation?: Navigation | undefined);
+}
+```
+
+`NavigationSyncer` is a `Syncer` that uses [Navigation API](https://github.com/WICG/navigation-api) to synchronize with history location. The constructor of `NavigationSyncer` must be passed an object equivalent to `window.navigation`.
+
+#### Parameters
+
+- `navigation?`: `window.navigation` or polyfill from [Navigation API](https://github.com/WICG/navigation-api).
+
+#### Example
+
+```tsx
+const navigationSyncer = new NavigationSyncer(
+  typeof window !== "undefined" ? window.navigation : undefined,
+);
+```
+
+#### `unsafeNavigation`
+
+For browsers that do not support the Navigation API, we provide an API called `unsafeNavigation`. This is a polyfill-like implementation that partially supports the behavior of the Navigation API, but the scope of implementation is minimal and **the `location-state` does not actively test and support it**.
```

Comment:
> the scope of implementation is minimal and **the `location-state` does not actively test and support it**.

これだけwarningにしてもいい気もするんですが、どうでしょう？

Reply:
いいですね！

Reply:
一回文章で加えてみましたがSSRの話を忘れてて、最終的には文章だとちょっと長くなりそうだったのでリストにしました。
dbfcd3d50364dc70ad0021b07d79b781d39166e3
f5241d470a748b7398b557857e02dd646accdea7

Reply:
リストが何なのか一言ほしいですね
たとえば

実際の値は実行環境に応じて以下となります
The actual value is below, depending on the runtime environment.

あとクライアントサイドの二つがどういう場合なのかあった方がよさげ

- Client side
  - `window.navigation` when the Navigation API is supported.
  - Otherwise, Navigation API polyfill (partially implemented).

Reply:
修正しました
87a971454900f938a29d43c6044782f18db1e827